app/models/pawn.rb

```diff
@@ -1,2 +1,8 @@
 class Pawn < Piece
+
+  def valid_move?(new_x, new_y)
+    x_dif = (new_x.to_i - x_position).abs
+    y_dif = (new_y.to_i - y_position).abs
+    (x_dif == 0) && (y_dif == 1)
```

Comment:
The pawn is a little bit tricky.. it can move 2 squares on its first move.. so we'd need to keep record if the piece has moved yet or not. Maybe adding a `has_moved` column to the db on pieces and setting it to false by default and once the piece moves updating it to true..

Reply:
this is what I came up with.  (Just easier to add gist I think) 

https://gist.github.com/PaleoGuy2012/da827f263283ef5d94416775ff019913

Reply:
Nice suggestions, @PaleoGuy2012!